example/src/main/java/com/stripe/example/activity/LauncherActivity.kt

```diff
@@ -121,6 +121,10 @@ class LauncherActivity : AppCompatActivity() {
                 activity.getString(R.string.netbanking_example),
                 NetbankingPaymentActivity::class.java
             ),
+            Item(
+                activity.getString(R.string.us_bank_account_example),
+                USBankAccountPaymentMethodActivity::class.java
```

Comment:
Maybe we can call this `ManualUSBankAccountPaymentMethodActivity` and mine `ConnectUSBankAccountPaymentMethodActivity`?